app/pathogen/arbovirus/analyze/recharts.tsx

```diff
@@ -0,0 +1,906 @@
+        NAb: antibody === "NAb" ? 1 : 0,
+        NR: antibody === "NR" ? 1 : 0,
+        "IgG, IgM": antibody === "IgG, IgM" ? 1 : 0,
+      });
+    }
+  });
+
+  return (
+    <ResponsiveContainer width={"100%"} height={"100%"}>
+      <BarChart width={730} height={250} data={data}>
+        <CartesianGrid strokeDasharray="3 3" />
+        <XAxis dataKey="arbovirus" />
+        <YAxis />
+        <Tooltip />
+        <Legend
+          layout="vertical"
+          verticalAlign="middle"
+          align="right"
+          wrapperStyle={{ right: -10 }}
+        />
+        <Bar dataKey="IgG" stackId="a" fill={"purple"} />
+        <Bar dataKey="IgM" stackId="a" fill={"blue"} />
+        <Bar dataKey="IgG, IgM" stackId="a" fill={"green"} />
+        <Bar dataKey="NAb" stackId="a" fill={"orange"} />
+        <Bar dataKey="NR" stackId="a" fill={"red"} />
+      </BarChart>
+    </ResponsiveContainer>
+  );
+}
+
+//Seroprevalence per pathogen, WHO region and age group
+// - Missing region data, need to calculate it, how?
+
+// Cumulative study count over time by pathogen
+
+interface StudyCountOverTime extends dataStratifiedByArbovirus {
+  year: number;
+}
+
+export function StudyCountOverTime() {
+  const state = useContext(ArboContext);
+
+  const data: StudyCountOverTime[] = [];
+
+  state.filteredData.forEach((d: any) => {
+    const year = new Date(d.sample_end_date).getFullYear();
+    let arbovirus: arboviruses = convertArboSFtoArbo(d.pathogen);
+
+    const existingData = _.find(data, { year: year });
+
+    if (existingData) {
+      existingData[arbovirus]++;
+    } else {
+      data.push({
+        year: year,
+        Zika: arbovirus === "Zika" ? 1 : 0,
+        Dengue: arbovirus === "Dengue" ? 1 : 0,
+        Chikengunia: arbovirus === "Chikengunia" ? 1 : 0,
+        "Yellow Fever": arbovirus === "Yellow Fever" ? 1 : 0,
+        "West Nile": arbovirus === "West Nile" ? 1 : 0,
+        Mayaro: arbovirus === "Mayaro" ? 1 : 0,
+      });
+    }
+  });
+
+  data.sort((a, b) => a.year - b.year);
+
+  for (let i = 1; i < data.length; i++) {
+    const element = data[i];
+
+    element.Dengue += data[i - 1].Dengue;
+    element.Zika += data[i - 1].Zika;
+    element.Chikengunia += data[i - 1].Chikengunia;
+    element["Yellow Fever"] += data[i - 1]["Yellow Fever"];
+    element["West Nile"] += data[i - 1]["West Nile"];
+    element.Mayaro += data[i - 1].Mayaro;
+  }
+
+  return (
+    <ResponsiveContainer width={"100%"} height={"100%"}>
+      <AreaChart
+        margin={{
+          top: 0,
+          right: 30,
+          left: 0,
+          bottom: 0,
+        }}
+        width={730}
+        height={250}
+        data={data}
+      >
+        <CartesianGrid strokeDasharray="3 3" />
+        <XAxis dataKey="year" />
+        <YAxis />
+        <Tooltip />
+        <Legend
+          layout="vertical"
+          verticalAlign="middle"
+          align="right"
+          wrapperStyle={{ right: -10 }}
+        />
+        <Area
```

Comment:
Similar kind of thing to my other comment on line 654 to line 782
We could have some kind of global object like this:
```
  const groupings = [
    {
      dataKey: "Zika",
      fillColor: pathogenColors.ZIKV,
    },
    {
      dataKey: "Dengue",
      fillColor: pathogenColors.DENV,
    },
    {
      dataKey: "Chikengunia",
      fillColor: pathogenColors.CHIKV,
    },
    {
      dataKey: "Yellow Fever",
      fillColor: pathogenColors.YF,
    },
    {
      dataKey: "West Nile",
      fillColor: pathogenColors.WNV,
    },
    {
      dataKey: "Mayaro",
      fillColor: pathogenColors.MAYV,
    },
  ] as const;
```
And make use of maps?
```
        {groupings.map(({ dataKey, fillColor }) => (
          <Area
            type="monotone"
            dataKey={dataKey}
            stackId="1"
            stroke={fillColor}
            fill={fillColor}
          />
        ))}
```